cpp/oneapi/dal/backend/primitives/reduction/test/reduction_rm_cw_perf_dpc.cpp

```diff
@@ -51,6 +51,15 @@ class reduction_rm_test_uniform : public te::float_algo_fixture<std::tuple_eleme
         CAPTURE(width_, stride_, height_);
     }
 
+    void generate_special() {
+        width_ = GENERATE(28, 960, 2000, 3072);
+        height_ = GENERATE(1024, 4096);
+        stride_ = height_;
+        SKIP_IF(width_ > stride_);
```

Comment:
This works only in TEST: `SKIP_IF` is just a macro that expands to `if (expr) return`. Suggest to add `width_ > stride_` check to `should_be_skipped`.

Reply:
Fixed